uobtheatre/payments/transaction_providers.py

```diff
@@ -587,6 +587,10 @@ def pay(
             currency=amount_details["currency"],
         )
 
+    @classmethod
+    def cancel(cls, payment: "payment_models.Transaction") -> None:
+        payment.delete()
```

Comment:
Should we have this? I don't think a square online payment can ever be canceled?